src/OperationHandler/UpdateHandler.php

```diff
@@ -113,4 +117,21 @@ public function getOutput(OperationInterface $operation, UrlGenerator $urlGenera
 
         return $output;
     }
+
+    /** @return string */
+    private function getSemverOutput(string $versionFrom, string $versionTo)
+    {
+        $versionsFrom = \explode('.', $versionFrom);
+        $versionsTo = \explode('.', $versionTo);
+
+        if (version_compare($versionsFrom[0], $versionsTo[0], '<')) {
```

Comment:
to better handle (uncommon) cases of downgrade, shouldn't we compare if major version are just different instead of one greater than the other?